### DISCUSSION TITLE
Problems starting Trilium in Manjaro (Arch) Linux

### DISCUSSION THREAD

**Discussion by Waxwood** [TriliumNext/Trilium#1462] [General]
Might you be able to include an build for Arch Linux? 

Currently, Arch users must rely on the community AUR builds which are often outdated. 
It seems a recent Manjaro (Arch) update broke my installation of Trilium 0.45.4-1. It is installed in /opt/trilium-bin/ and will not start. The AUR version has not been updated to 0.45.5.

I manually installed 0.45.5 to /home/USER/trilium/trilium-linux-x64 but for some reason can only run portable-trilium.sh but it starts with the default database.

I confirmed that my database located in /home/USER/.local/share/trilium-data and followed the instructions on the Data Directory wiki entry and used this command to try to point Trilium to my database directory 
`export TRILIUM_DATA_DIR=/home/USER/trilium-data`
but must have done something wrong. Trilium is not able to load 

For convenient backup, I attempted to move database to a location I created on HOME 

`export TRILIUM_DATA_DIR=/home/USER/trilium-data`
But the data did not move there

Then I attempted to point Trilum to the current location of the database

`export TRILIUM_DATA_DIR=/home/USER/.local/share/trilium-data`
But the portable version was still unable to access the database.

Finally, following the instructions on the wiki entry, Data Directory, to point the data directory to Portable Trilium, I tried this:

`TRILIUM_DATA_DIR=/home/USER/.local/share/trilium-data trilium`

This finally worked. Trilium portable is now able to open my database.

I still have no idea however how to move my database into /home/USER/trilium-data where its easier to keep track of and backup.

It seems that many new users will not be able to figure out this convoluted process if Trilium fails to start after a system update. 
For this reason, please consider adding an ARCH build. 

Also, more importantly, please allow the user to select the Trlium database at startup, and through the system menu. 

Thank you for your outstanding work on Trilium. If your work continues at this fast pace, Trilium's feature set will overtake even the best of the note-taking applications and become an outstanding tool for researchers worldwide.

**Comment by zadam** [comment]
> I still have no idea however how to move my database into /home/USER/trilium-data where its easier to keep track of and backup.

Hi, `mv` should do the trick, hm? I'm sort of confused, as an arch user you're probably not a linux beginner ...

> For this reason, please consider adding an ARCH build.

Sorry, but this is not going to happen. I have no knowledge of Arch and supporting all the possible distros would be for me huge time sink.

**Reply by Waxwood** [reply]
> Hi, `mv` should do the trick, hm? I'm sort of confused, as an arch user you're probably not a linux beginner ..

Haha.. yes, good point! But I was concerned that moving it would cause some other problem- I don't understand how Trilium is linked to the database and not wanting to complicate things more, I did not move it. I used Manjaro not because I'm an advanced user, but because Manjaro is far less trouble than other distros I've used in the past, runs fast, and small problems are easy to fix. 

> Sorry, but this is not going to happen. I have no knowledge of Arch and supporting all the possible distros would be for me huge time sink.
Yes, of course, in hindsight I can see it would be impossible to keep up. Thank you for your work and taking the time to reply.

**Reply by zadam** [reply]
You can move it to ~/trilium-data and nothing should break. Trilium just looks for standard directories (/home/USER/.local/share/trilium-data is the default).

If you move it to ~/trilium-data, then you shouldn't even need to set the env. variable since ~/trilium-data is used as a fallback path when the default /home/USER/.local/share/trilium-data is not found.

**Reply by Waxwood** [reply]
Excellent! Everything is working well now. Though I still can't get the Docker version to work, I'll eventually figure it out. Thank you again zadam!
